File: src/Interfaces/EAccuracy.java
package Interfaces;

public enum EAccuracy {
    SAMPLE_LOW(1000),
    SAMPLE_HIGH(10000),
    SAMPLE_OVERHEAT(100000),
    SAMPLE_WTF(1000000);

    public double size;

    EAccuracy(double s) {
        size=s;
    }
}

File: src/Interfaces/EAccuracy.kt
package Interfaces

enum class EAccuracy private constructor(var size: Double) {
    SAMPLE_ROBLOX(100.toDouble()),
    SAMPLE_LOW(1000.toDouble()),
    SAMPLE_HIGH(10000.toDouble()),
    SAMPLE_OVERHEAT(100000.toDouble()),
    SAMPLE_WTF(1000000.toDouble())
}
